### ISSUE TITLE
Promote sharing an issue more

### ISSUE THREAD

**Issue by agjohnson** [5calls/5calls#178]
So, though the primary motivation is to get users calling, getting others involved in calling could be the secondary goal here. Getting a regular user to onboard new users, especially users in conservative areas, is almost as valuable as that user spending time to call -- or perhaps more valuable if that user's reps are already progressive.

This could certainly increase complexity on the backend, where currently issues are filtered out if they aren't directly actionable, and would require frontend changes, where perhaps the action isn't just assumed to be a call.

There are two cases where users could be directed more aggressively to share the issue:

## After calling

Users are already told that they can share the issue at this point, perhaps this needs to be a more convincing. This is the simplest change that can be made. The post-call interface could instead tell users about the impact of getting people in other states involved, etc, and perhaps even some form of dynamic reason -- ie: "if you have friends in these states, they should get involved on this issue: ..."

## Not a constituent

A more important change would be giving users who are not a constituent, or can't otherwise directly affect an issue, a discrete task. If you are a user affected by an issue that you can't take action on, you still might have contacts in areas that can. Instead of a call, perhaps these users are asked to spread knowledge of the issue?

**Comment by nickoneill** [comment]
This makes a lot of sense, particularly in the case of people looking for something to do after calling. Or just in general, getting users to onboard other users.

I'm not sure about the backend complexities you mention. How would this have an impact there?

**Comment by nickoneill** [comment]
🏓

**Comment by agjohnson** [comment]
To implement a stronger call follow up sharing page, each issue could also store and return through API suggestions for sharing the issue. Perhaps for simplicity's sake, this field is just a blob of text? Changes for this feature would be pretty minimal: return an extra field from the issue api, and the frontend would selectively show that text on the follow up page if it exists.

There are probably a few ways to implement the second part of this, the two I'm thinking of:

* Issues that you can't act on are flagged on the issue api return. This flag signals that the front end should display sharing information instead of the normal call view.
* Create another target for the database entires, `EMAIL`, for the backend to filter against. Authors to selectively pick issues to promote to share. This would increase the burden on authoring issues though, as it might duplicate work.

The only complexity I'd be worried about introducing with altering the issue API return is the effect on android/ios clients -- I haven't followed development of these however.

**Comment by nickoneill** [comment]
Yeah, at that level of detail this could become a larger backend task. But we essentially have a single class of issue right now: something that is applicable nationally. A better promotion here doesn't have to be complex, you can send it to nearly anyone.

A few things do drive shares more strongly though: counts related to how others have shared this issue (like the standard twitter and facebook buttons) and a custom social blurb, which could come from the backend. Right now we just do some dumb "I just called on [insert title here] and you should too"
